frontend/device/remarkable/device.lua

```diff
@@ -17,39 +29,61 @@ local Remarkable = Generic:new{
     display_dpi = 226,
     -- Despite the SoC supporting it, it's finicky in practice (#6772)
     canHWInvert = no,
-    home_dir = "/mnt/root",
+    home_dir = "/home/root",
```

Comment:
This is for both devices?

Reply:
yes, the have the same file system layout